Resources/views/ContentType/update_content_type.html.twig

```diff
@@ -25,72 +27,101 @@
 
 {% block content %}
     <section class="ez-serverside-content">
-        Last modified: {{ contentTypeDraft.modificationDate|localizeddate('medium', 'medium', app.request.locale) }}
 
-        {{ form_start(form, {"action": action_url}) }}
+        <div class="context-information">
```

Comment:
Found `ez-technical-infos`, won't apply here now but adding it as it is the closest fit (used in content edit view atm).